include/zephyr/net/net_ip.h

```diff
@@ -1239,6 +1239,39 @@ static inline void net_ipv6_addr_create_ll_allrouters_mcast(struct in6_addr *add
 	net_ipv6_addr_create(addr, 0xff02, 0, 0, 0, 0, 0, 0, 0x0002);
 }
 
+/**
+ *  @brief Create IPv4 mapped IPv6 address
+ *
+ *  @param addr4 IPv4 address
+ *  @param addr6 IPv6 address to be created
+ */
+static inline void net_ipv6_addr_create_v4_mapped(const struct in_addr *addr4,
+						  struct in6_addr *addr6)
+{
+	net_ipv6_addr_create(addr6, 0, 0, 0, 0, 0, 0xffff,
+			     ntohs(addr4->s4_addr16[0]),
+			     ntohs(addr4->s4_addr16[1]));
+}
+
+/**
+ *  @brief Is the IPv6 address an IPv4 mapped one. The v4 mapped addresses
+ *         look like ::ffff:a.b.c.d
```

Comment:
The issue is with Sphinx, that thinks it's a directive. The following should do the trick (and it's not a bad idea anyway to format the IP address as code, I guess?)

```suggestion
 *         look like `::ffff:a.b.c.d`
```

Reply:
I figured it out, using `\` worked
```
diff --git a/include/zephyr/net/net_ip.h b/include/zephyr/net/net_ip.h
index 20d95d68b8..bc760e7323 100644
--- a/include/zephyr/net/net_ip.h
+++ b/include/zephyr/net/net_ip.h
@@ -1255,7 +1255,7 @@ static inline void net_ipv6_addr_create_v4_mapped(const struct in_addr *addr4,
 
 /**
  *  @brief Is the IPv6 address an IPv4 mapped one. The v4 mapped addresses
- *         look like `::ffff:a.b.c.d`
+ *         look like \::ffff:a.b.c.d
  *
  *  @param addr IPv6 address
  *
```